src/Components/NavBar.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+import { Link } from 'react-router-dom';
+import '../Styling/Navbar.css';
+import { MenuItems } from "./MenuItems";
+import { HiOutlineX } from "react-icons/hi";
+import SubMenu from './SubMenu';
+
+
+
+function Navbar() {
+    const [sidebar, setSidebar] = useState(false);
```

Comment:
personally, I think lines 11-14 would look cleaner like this:

   const [sidebar, setSidebar] = useState(false);
   const showSidebar = () => setSidebar(!sidebar);

     return( 
     ...